_posts/2017-5-31-release-1.3.0.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+title:  "Apache Flink 1.3.0 Release Announcement"
+date:   2017-05-31 19:00:00
+author: "Robert Metzger"
+author-twitter: "rmetzger_"
+categories: news
+---
+
+
+The Apache Flink community is pleased to announce the 1.3.0 release. Over the past 4 months, the Flink community has been working hard to resolve more than 680 issues. See the [complete changelog]() for more detail.
+
+This is the fourth major release in the 1.x.y series. It is API compatible with the other 1.x.y releases for APIs annotated with the @Public annotation.
+
+We encourage everyone to download the release and check out the [documentation](https://ci.apache.org/projects/flink/flink-docs-release-1.3/). Feedback through the [Flink mailing lists](http://flink.apache.org/community.html#mailing-lists) is, as always, gladly encouraged!
+
+You can find the binaries on the updated [Downloads page](http://flink.apache.org/downloads.html). Some highlights of the release are listed below.
+
+
+# Large State Handling / Recovery
```

Comment:
I think we should remove the whitespace before and after `/`

https://en.wikipedia.org/wiki/Slash_(punctuation)#Spacing